src/utilities/testing/render.tsx

```diff
@@ -1,13 +1,13 @@
 jest.mock('../../utilities/api/interactionManager');
 jest.mock('react-router-dom', () => ({
-  ...jest.requireActual('react-router-dom'),
+  ...(jest.requireActual('react-router-dom') as typeof ReactRouterDom),
```

Comment:
I'm cool to check these in as a fix for that error -- but do want to check with @bvillaroman. We can also get these in separately if we're feeling low-confidence about them.

Reply:
ah damn typescript lol. Yeah this looks good for me

Reply:
thanks bruce